rocketpy/simulation/flight.py

```diff
@@ -24,8 +24,19 @@
     quaternions_to_spin,
 )
 
+ODE_SOLVER_MAP = {
+    'RK23': RK23,
+    'RK45': RK45,
+    'DOP853': DOP853,
+    'Radau': Radau,
+    'BDF': BDF,
+    'LSODA': LSODA,
+}
```

Comment:
Could this possible become an Enum? Seems more suitable

Reply:
Certainly could be an Enum, however I prefer keeping it like this to better mirror [`scipy` usage](https://github.com/scipy/scipy/blob/v1.14.1/scipy/integrate/_ivp/ivp.py#L12-L17), I could even have imported their dictionary directly, but that is not guaranteed to not have breaking changes, so I preferred to simply build the dict myself.